docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,377 @@
+list of available databases:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+$ tsh db ls
+# Name      Description         Labels
+# --------- ------------------- -------
+# sqlserver                     env=dev
+```
+
+</ScopedBlock>
+
+To retrieve credentials for a database and connect to it:
+
+```code
+$ tsh db connect --db-user=teleport sqlserver
+```
+
+<Admonition type="note">
+  The `mssql-cli` command-line client should be available in `PATH` of the machine
+  you're running `tsh db connect` from.  
+  
+  `mssql-cli` is not required for SQL Server connections.  Use `tsh proxy db --db-user=teleport --tunnel sqlserver`
+   to connect from other DB Clients such as Microsoft SQL Server Management Studio.
+</Admonition>
+
+To log out of the database and remove credentials:
+
+```code
+$ tsh db logout sqlserver
+```
+
+## Troubleshooting
+
+### Teleport CA and CRL not imported correctly.
+
+When connecting to your database, you get an error `Error message: authentication
+failed` and Teleport database service logs have the error message `Failed to
+authenticate with KDC: kinit: Client not trusted while getting initial
+credentials`. This happens when the Teleport Database CA is not imported
+correctly or propagated yet. You can force the propagation by running
+`certutil -pulse` and trying to connect to your database.
+
+### Invalid KDC hostname.
+
+If you’re connecting to your database and receive the error `Error message:
+authentication failed` and on Teleport database service logs, there is the error
+entry `Failed to authenticate with KDC: Password for user@AD.TELEPORT.DEV:
+\nkinit: Cannot read password while getting initial credentials`, which means
+that the KDC hostname is wrong. You can verify your domain controller’s SPN to
+see if they’re set correctly and update the value on the field `kdc_hostname` on
+your database's configuration.
+
+### Teleport cannot verify database CA.
+
+If your database has a CA that Teleport doesn’t know about, it will return the
+following error when connecting to it: `Error message: TLS Handshake failed:
+x509: certificate signed by unknown authority (possibly because of "x509:
+invalid signature: parent certificate cannot sign this kind of certificate"
+while trying to verify candidate authority certificate
+"SSL_Self_Signed_Fallback")`.
+
+To solve this, you can add the CA configuration to the database like the
+following:
+
+```yaml
+...
+db_service:
+  databases:
+    - name: sqlserver
+      protocol: sqlserver
+      tls:
+        # Point it to your Database CA PEM certificate.
+        ca_cert_file: "rdsca.pem"
```

Comment:
Does this is enough. Most likely SQLServer cert has empty `CN` SANs field and switching to `mode: verify-ca` from ` verify-full` will be needed in most cases.